apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestDataPathsActivity.java

```diff
@@ -79,7 +76,8 @@ public class TestDataPathsActivity  extends BaseAutoGlitchActivity {
     private double mMaxMagnitude;
     private int    mPhaseCount;
     private double mPhase;
-    private double mPhaseJitter;
+    private double mPhaseErrorSum;
+    private double mPhaseErrorCount;
```

Comment:
nit: mPhaseErrorCount is a confusing variable name as it makes you think this is the count of the number of phase errors. I can't think of a much better name though

Reply:
I agree that Error has bad connotations.
And I also cannot think of a better name.